wmfdata/spark.py

```diff
@@ -74,22 +75,30 @@
     'no_proxy',
 ]
 
-def get_custom_session(
+def get_active_session():
+    """
+    Returns the active session if there is one and None otherwise.
+
+    Spark 3 includes a native getActiveSession function. Once we end Spark 2
+    support, we can switch to that.
+    """
+    if SparkContext._jvm:
+        if SparkContext._jvm.SparkSession.active():
+            return SparkSession.builder.getOrCreate()
```

Comment:
I just fixed `get_active_session` so that it doesn't throw an error when the most recent session was stopped. While I was added, I made it the return explicitly in all cases